apple-codesign/tests/cmd/windows-store-export-certificate-chain.trycmd

```diff
@@ -0,0 +1,14 @@
+```
+$ rcodesign help windows-store-export-certificate-chain
+Export CA certificates from the Windows Store
+
+Usage: rcodesign.exe windows-store-export-certificate-chain [OPTIONS] --thumbprint <THUMBPRINT>
```

Comment:
This needs to be `Usage: rcodesign[EXE] ...` to avoid failing tests on Linux and macOS.